spec/features/course/assessment_condition_management_spec.rb

```diff
@@ -0,0 +1,182 @@
+require 'rails_helper'
+
+RSpec.feature 'Course: Assessments' do
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
```

Comment:
@minhtule This looks fine, but maybe since this is repeated there is a way to just check the include, there's no need to test every single combination.